vagrant/provisioning/roles/elasticsearch-install/templates/elasticsearch.yml

```diff
@@ -0,0 +1,14 @@
+cluster.name: {{ cluster_name | default('acm-cluster') }}
+node.name: {{ node_name | default('acm-node01') }}
+path.data: {{ root_folder }}/data/elasticsearch
+path.logs: {{ root_folder }}/log/elasticsearch
+#path.conf: /etc/elasticsearch
+network.host: localhost
+http.port: {{ ES_PORT | default(9200) }}
```

Comment:
We don't use upper case naming of variables. I would name the variable with longer name **elasticsearch_port**.

Reply:
@bojmil-armedia , since this role is only for installation and basic configuration of elasticsearch service, i think it would be nice if it remains with the name elasticsearch-install. Also, i guess there will be another task for configuration which will be wrapped into the new role 'elasticsearch-configure'. For me, there is no problem to change it